build.gradle

```diff
@@ -184,3 +184,14 @@ configure(allprojects - project(':conductor-grpc')) {
         }
     }
 }
+
+["cassandra-persistence", "core", "redis-concurrency-limit", "test-harness"].each {
```

Comment:
Should this configuration be done for all projects? We will end up adding Spock tests for most of the modules.

Reply:
we can extend this to all java projects, i only added the ones that currently have spock tests